index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="stylesheet" href="style.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,500;0,600;0,800;1,100&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <nav>
+            <!-- <img src="picture/header-shapes mobile.png" alt=""> -->
+
+            <div class="logo">
+                <p>Logo</p>
+            </div>
+    
+             <div class="burger">
+                    <img src="picture/Enabled.png" alt="">
+                  </div>
+        </nav>
+
+    </header>
+        <section>
+
+            <nav class="headline">
+                <!-- <img src="picture/header-shapes mobile.png" alt=""> -->
+                <h1>I'm Steven. Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <h2>LET’S CONNECT</h2>
+                <div class="Icon">
+                    <a href="https://twitter.com/steven_hakeem">
+                        <i class="fa-brands fa-twitter"></i>
+                        </a>
+                    <a href="https://www.linkedin.com/in/steven-wafeek-b629341b6/">
+                        <i class="fa-brands fa-linkedin-in"></i>
+                        </a>
+                        <a href="https://twitter.com/steven_hakeem">
+                             <i class="fa-brands fa-github"></i>
+                            </a>
+                            <a href="#">
+                        <i class="fa-regular fa-hand-peace fa-flip-horizontal"></i>
+                    </a>
+                </div>
```

Comment:
- [ ] Kindly implement the list of social icons to be in a list instead, you can use something like the following:
```
<ul>
   <li>
      <a>
         <i>...</i>
      </a>
   </li>
   <li>
      <a>
         <i>...</i>
      </a>
   </li>
   <li>
      <a>
         <i>...</i>
      </a>
   </li>
   ...
</ul>
```